app/Models/User.php

```diff
@@ -2,11 +2,14 @@
 
 namespace App\Models;
 
+use Illuminate\Auth\Notifications\ResetPassword as ResetPasswordNotification;
+use Illuminate\Contracts\Auth\CanResetPassword;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Notifications\Notifiable;
 
-class User extends Model {
-    use SoftDeletes;
+class User extends Model implements CanResetPassword {
```

Comment:
I suggest using  `Illuminate\Foundation\Auth\User` which automatically implement the following interfaces

- CanResetPasswordContract
- AuthenticatableContract : Get the unique identifier for the user when routes are protected with **auth** middleware
- AuthorizableContract : Implement abilities system